flexget/components/ftp/sftp_client.py

```diff
@@ -169,21 +178,25 @@ def download(self, source: str, to: str, recursive: bool, delete_origin: bool) -
         if not self.path_exists(source):
             raise SftpError(f'Remote path does not exist: {source}')
 
+        is_symlink: bool = self.is_link(source)
         if self.is_file(source):
             source_file: str = parsed_path.name
             source_dir: str = parsed_path.parent.as_posix()
             try:
                 self._sftp.cwd(source_dir)
-                self._download_file(to, delete_origin, source_file)
+                self._download_file(to, delete_origin and not is_symlink, source_file)
```

Comment:
I'd consider binding `delete_origin and not is_symlink` to a named variable to make it clearer what the combined value means.